Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -3862,27 +3862,26 @@ extension PaymentSheetUITestCase {
     }
 
     func testPayingWithNoFormPMs_verticalmode() {
-        // We choose Cash App as a representative PM that does not require form details
+        // We choose Alipay as a representative PM that does not require form details
         var settings = PaymentSheetTestPlaygroundSettings.defaultValues()
         settings.mode = .payment
         settings.layout = .vertical
         loadPlayground(app, settings)
 
-        // Try Cash App
+        // Try Alipay
```

Comment:
Switched to Alipay due to https://github.com/stripe/stripe-ios/pull/3661, otherwise we'd need to wait 15 seconds for polling to timeout.

Reply:
Is that a new change? Cash App canceled immediately for me.

Reply:
Yes new, CAP was mistakenly not addd to this list when we disabled refreshing for it.